README.rst

```diff
@@ -8,6 +8,17 @@ Basic Redis locking mechanism, implemented as an asynchronous context manager. A
 * Free software: MIT
 * Documentation: https://redis-heartbeat-lock.readthedocs.io.
 
+Sponsor
+--------
```

Comment:
Huh, does Python default to `*.rst` files instead of Markdown? I'm not really sure what using a different markup language actually buys that's worth the effort of remembering slightly different rules.

Reply:
Well, they're autogenerated by the package I used to create this, and Python read the docs does default to `.rst`.